src/parameters/create_parameters.jl

```diff
@@ -10,10 +10,11 @@ import StaticArrays as SA
 function ClimaAtmosParameters(config::AtmosConfig)
     (; toml_dict, parsed_args) = config
     FT = CP.float_type(toml_dict)
-    return ClimaAtmosParameters(
-        toml_dict,
-        FT(CA.time_to_seconds(parsed_args["dt"])),
-    )
+    override_dt =
+        parsed_args["override_precip_timescale"] ?
+        FT(CA.time_to_seconds(parsed_args["dt"])) : nothing
+
+    return ClimaAtmosParameters(toml_dict, override_dt)
 end
```

Comment:
Should we move the function after the doc string?

Reply:
yes, thank you!